.github/workflows/build.yml

```diff
@@ -2,7 +2,7 @@ name: Build
 
 on:
   push:
-  pull_request:
```

Comment:
Was this intentional? The CI passing/failing feedback is very useful in PRs, imo.